aws/table_aws_wellarchitected_answer.go

```diff
@@ -0,0 +1,317 @@
+		return nil, err
+	}
+
+	// Unsupported region, return no data
+	if svc == nil {
+		return nil, nil
+	}
+
+	for _, lensAlias := range workload.Lenses {
+		if d.EqualsQualString("lens_alias") != "" && d.EqualsQualString("lens_alias") != lensAlias {
+			continue
+		}
+		if d.EqualsQualString("pillar_id") != "" {
+			input.PillarId = aws.String(d.EqualsQualString("pillar_id"))
+		}
+		if d.EqualsQuals["milestone_number"] != nil {
+			input.MilestoneNumber = int32(d.EqualsQuals["milestone_number"].GetInt64Value())
+		}
+		input.LensAlias = aws.String(lensAlias)
+		input.WorkloadId = aws.String(*workload.WorkloadId)
+
+		paginator := wellarchitected.NewListAnswersPaginator(svc, input, func(o *wellarchitected.ListAnswersPaginatorOptions) {
+			o.Limit = maxLimit
+			o.StopOnDuplicateToken = true
+		})
+
+		// List call
+		for paginator.HasMorePages() {
+			output, err := paginator.NextPage(ctx)
+			if err != nil {
+				if strings.Contains(err.Error(), "ResourceNotFoundException") || strings.Contains(err.Error(), "ValidationException") {
+					return nil, nil
+				}
+				plugin.Logger(ctx).Error("aws_wellarchitected_answer.listWellArchitectedAnswers", "api_error", err)
+				return nil, err
+			}
+
+			for _, item := range output.AnswerSummaries {
+
+				answer := types.Answer{
+					Choices:       item.Choices,
+					IsApplicable:  item.IsApplicable,
+					PillarId:      item.PillarId,
+					QuestionId:    item.QuestionId,
+					QuestionTitle: item.QuestionTitle,
+					Reason:        item.Reason,
+					Risk:          item.Risk,
+				}
+
+				d.StreamListItem(ctx, &AnswerInfo{answer, output.LensAlias, output.LensArn, &output.MilestoneNumber, output.WorkloadId})
+
+				// Context can be cancelled due to manual cancellation or the limit has been hit
+				if d.RowsRemaining(ctx) == 0 {
+					return nil, nil
+				}
+			}
+		}
+	}
+
+	return nil, nil
+}
+
+//// HYDRATE FUNCTIONS
+
+func getWellArchitectedAnswer(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	var questionId, lensAlias, workloadId string
+	var milestoneNumber int32
+	if h.Item != nil {
+		answer := h.Item.(*AnswerInfo)
+		questionId = *answer.QuestionId
+		lensAlias = *answer.LensAlias
+		workloadId = *answer.WorkloadId
+		milestoneNumber = *answer.MilestoneNumber
+	} else {
+		questionId = d.EqualsQualString("question_id")
+		lensAlias = d.EqualsQualString("lens_alias")
+		workloadId = d.EqualsQualString("workload_id")
+		if d.EqualsQuals["workload_id"] != nil {
+			milestoneNumber = int32(d.EqualsQuals["workload_id"].GetInt64Value())
+		}
+	}
+
+	// Validate - User inputs must not be blank
+	if questionId == "" || lensAlias == "" || workloadId == "" {
+		return nil, nil
+	}
+
+	params := &wellarchitected.GetAnswerInput{
+		QuestionId: aws.String(questionId),
+		LensAlias:  aws.String(lensAlias),
+		WorkloadId: aws.String(workloadId),
+	}
+	if milestoneNumber != 0 {
+		params.MilestoneNumber = milestoneNumber
+	}
```

Comment:
What occurs if the user does not provide the `MilestoneNumber` parameter? Will the API return an error? Also is it a required parameter for making the API call?

I can't see `milestone_number` column has been added in `GetConfig`, do we need these lines of code?